src/effects/native/flangereffect.h

```diff
@@ -11,17 +11,35 @@
 #include "util/sample.h"
 #include "util/types.h"
 
+namespace {
+constexpr double kMaxDelayMs = 13.0;
+constexpr double kMinDelayMs = 0.22;
```

Comment:
Where did these numbers come from? Are they arbitrary values you determined by guessing and checking what sounds good? It is fine if that is the case as long as you document it in a comment.

Reply:
I am curious to read @sohet results.
Hardware Flangers start at 0.4 or 0.3. to 13 or 14.  I pick 0.22 because that's our old minimum at 44.1 kHz. It was 0.1 at @96 kHz though.